content/blog/2023-04-13-rse-on-mastodon.md

```diff
@@ -0,0 +1,78 @@
++++
+title = "State of Research software engineers on Mastodon"
+
+[extra]
+
+author = "Richard Darst"
++++
+
+Many domains have migrated to Mastodon as of 2023.  Have Research
+Software Engineers?  This post examines the situation, based on [this
+post by rkdarst](https://fosstodon.org/@rkdarst/110146601969193973)
+
+Summary: not much yet
+
+This post can be updated as we get more information/the situation
+evolves, please send pull requests.
+
+
+## RSE organizations
+
+* US RSE: <https://fosstodon.org/@us_rse>
+* DE RSE: <https://fosstodon.org/@de_rse@mastodon.social>
+* Nordic RSE: <https://fosstodon.org/@nordic_rse>
+* Society of RSE (UK/int): ??
+* ... ?
+
+
+## Other accounts to follow to seed your community
+
+TODO (should we join some other list instead of make a new one here?)
```

Comment:
I think getting a category at <https://fedi.directory/> as you mention below is the best option for this. It will be inherently more visible than any bespoke list we might start, and it avoids any decisions about where such a list would be hosted and maintained.  (This is @mattasdata from Mastodon btw. Need to reconcile my different usernames :stuck_out_tongue: )

Reply:
Does this mean removing this section for now, or keeping the TODO?